### PR TITLE
Assignment2 Calculator submission.

### PR DESCRIPTION
**Pull request by joeriddles**
CSCD371 Assignment 2 submission.

### CODE REVIEW

File: Assignment2/src/Calculator.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+			}
+
+			expressionOperator = "";
+			return false;
+		}
+
+		static bool InvalidExpression(string consoleOutput)
+		{
+			Console.WriteLine(consoleOutput);
+			return false;
```
Comment (Keboo) [inline Assignment2/src/Calculator.cs]: Why have a method that always returns a constant value? This appears to be something that might have been better inlined above.

**Comment by Keboo** [comment]
Nice work, however I do not see the string unit tests that were apart of the assignment.

File: Assignment2/test/CalculatorTest.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		[TestMethod]
+		public void TestCalculatorMultiplyAltSucceed()
+		{
+			ConsoleAssert.Expect(GenerateConsoleOutputSucceed(3, 5, "x", 15), Calculator.Calculator.Main);
+		}
+
+		[TestMethod]
+		public void TestCalculatorDivideSucceed()
+		{
+			ConsoleAssert.Expect(GenerateConsoleOutputSucceed(3, 5, "/", (decimal) 0.6), Calculator.Calculator.Main);
```
[the remark this answers is not in the document]
Comment (joeriddles) [reply Assignment2/test/CalculatorTest.cs]: I forgot about the literal notation. I'll keep that in mind moving forward working with doubles and decimals. Thanks.

File: Assignment2/src/Calculator.cs (hunk 1 above)
Comment (joeriddles) [reply Assignment2/src/Calculator.cs]: Good point. The thinking behind this was to shave off a line of code everywhere `InvalidExpression` was used. Looking back, I agree that inlined would read better.

File: Assignment2/src/Calculator.cs [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	public class Calculator
+	{
+		public static void Main(string[] args)
+		{
+			string userInput = "";
+
+			if (args.Length > 1)
+			{
+				Console.WriteLine($"Passed in expression: {String.Join(" ", args.Skip(1))}");
+				userInput = String.Join(" ", args.Skip(1));
```
[the remark this answers is not in the document]
Comment (joeriddles) [reply Assignment2/src/Calculator.cs]: I figured out what was going on here. I was under the impression to run my program from the command line, I had to type `dotnet run Calculator.cs 1 + 1`, but `Calculator.cs` was just being passed in as a string parameter. I've removed the code for skipping `args[0]`.

**Comment by joeriddles** [comment]
@Keboo, whoops, you are correct that I did not have the string unit tests. I glanced over that part when doing the assignment. I'm adding the string unit tests to my branch now. I realize it is now past the due date, but I'd appreciate it if you would take a peek at the unit tests if you get a chance.

**Comment by joeriddles** [comment]
Thanks for the feedback. My most recent commit addresses the comments you had.

**Pull request closed without merging**